src/Microsoft.ApplicationInsights.AspNet/ExceptionTrackingMiddleware.cs

```diff
@@ -13,11 +14,13 @@ public sealed class ExceptionTrackingMiddleware
     {
         private readonly RequestDelegate next;
         private readonly TelemetryClient telemetryClient;
+        private readonly string sdkVersion;
 
         public ExceptionTrackingMiddleware(RequestDelegate next, TelemetryClient client)
         {
             this.next = next;
             this.telemetryClient = client;
+            this.sdkVersion = SdkVersionUtils.VersionPrefix + SdkVersionUtils.GetAssemblyVersion();
```

Comment:
The idea here is to count how many people are using auto-collection, not making TrackException manually